lib/MetaCPAN/Web/Controller/Sysadmin/Datacenters.pm

```diff
@@ -0,0 +1,29 @@
+package MetaCPAN::Web::Controller::Sysadmin::Datacenters;
+
+use Moose;
```

Comment:
Ahh, no I started this in mc-api and that uses `Moose` not `MetaCPAN::Moose` - I didn't notice when I moved it over (mc-api is to ES specific!)